source/chapters/djing_with_mixxx.rst

```diff
@@ -401,9 +401,19 @@ Auto DJ
 
    Mixxx library - Auto DJ view
 
-Auto DJ allows you to automatically load tracks from the Auto DJ playlist when
-the current track is nearly finished, and crossfade into it.  See
-:ref:`library-auto-dj`.
+Auto DJ is the automatic Mixxx player engine. It automatically loads tracks in the decks and mixes them automatically.
+Auto DJ needs tracks, these tracks can be manually selected on track level, on crate level or automatic from the library, read more below.
+
+To be able to mix the tracks Auto DJ takes control of the crossfader. See crossfader.
+Auto DJ requires the allocation of at least 1 deck to each side of the crossfader. The assignment of the decks is done by the orientation-knobs. Each deck can be orientated to the left side, right side or center of the crossfader. If a deck's orientation is centered, the crossfader can't affect it.
+When Auto DJ is toggled on, Auto DJ wil search for 2 decks that are on a opposit side of the crossfader (not centered). Auto DJ will check the decks in numeric order: if deck 1 is left orientated Auto DJ wil continue to search for a right-orientated deck. If this is deck 2, Auto DJ will work with decks 1 & 2. If deck 1 is right-roientated (fi in hamster-mode) Auto DJ wil search for a left-orientated desck. This can be deck 2,3 or 4.
+If no opposed decks were found Auto DJ won't start.
+Even decks that aren not visually shown in your Mixxx-skin can be used. To show deck 3 & 4, you can toggle the 4 Decks-button on top in the user interface or in the skin settings
+See skin settings.
+The crossfader indicates which deck is playing and moves to the other side during the transition to the new track.
+When you toggle 'off' the Auto DJ, the crossfader will stay on it's last position.
+Since version 2.4.... you can choose to reset the crossfader to the centered positionafter disabling AUTO DJ by selecting the option 'reset crossfader to center after disabling Auto DJ' in the preferences. See :[crossfader](https://manual.mixxx.org/2.4/en/chapters/user_interface.html#crossfader](https://manual.mixxx.org/2.4/en/chapters/preferences).
+If Auto DJ is enabled, you can see a colored rectangule around the crossfader.
```

Comment:
```suggestion

When Auto DJ is toggled on, it will use the first two opposing decks it finds. With default settings these would be decks 1 and 2, but depending on how the orientation switches are set, this could be any configuration, for example deck 4 on the left and deck 1 on the right. If no opposing decks are found Auto DJ won't start.
Note that Auto DJ is currently not aware which decks are currently visible, i.e. it might pick currently hidden decks 3 and/or 4. Refer to the :ref:`skin settings <interface-skin-settings>` for how to toggle 4 decks.

When you toggle 'off' the Auto DJ, the crossfader will stay on it's last position.

Each official Mixxx skin has an Auto DJ indicator icon, located either in the top toolbar or in the mixer. In some skins the crossfader is highlighted, too.
```